morpho/processors/IO/IOROOTProcessor.py

```diff
@@ -64,16 +64,17 @@ def Reader(self):
                 import ROOT
             except ImportError:
                 logger.warning("Failed importing ROOT")
-                pass
-            infile = ROOT.TFile(self.file_name,"READ")
-            tree = infile.Get(self.tree_name)
-            tree.GetEntry(0)
-            for varName in self.variables:
-                value = getattr(tree, varName)
-                if type(value)==int or type(value)==float:
-                    value = [value]
-                self.data.update(
-                    {str(varName): self.data[str(varName)] + value})
+            else:
+                infile = ROOT.TFile(self.file_name,"READ")
+                tree = infile.Get(self.tree_name)
+                for i in range(0, tree.GetEntries()):
+                    tree.GetEntry(i)
+                value = []
+                for varName in self.variables:
+                    for j in range(0, tree.GetEntries()):
+                        value.append(getattr(tree, varName))
+                    self.data.update(
+                        {str(varName): self.data[str(varName)] + value})
```

Comment:
@guiguem, I changed this code determining how data gets read from a ROOT file – because before, when running the CalibrationProcessor, only the first entry of the array in a root tree would get read into python.

I think we may not have noticed this issue before because it only happens when using uproot fails.

Does the revised code look okay?